sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.10.0 (2020-10-28)
+## 1.10.0 (2020-10-29)
```

Comment:
Why did this change, this was still shipped yesterday?

Reply:
Didn't ship, had to make a few API changes last night